dfc/proxy.go

```diff
@@ -1613,6 +1626,32 @@ func (p *proxyrunner) tokenhdlr(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handler for dfc being used as a HTTP proxy
+func (p *proxyrunner) proxyhdlr(w http.ResponseWriter, r *http.Request) {
+	baseURL := r.URL.Scheme + "://" + r.URL.Host
+	switch baseURL {
+	case gcpApiUrl:
+		switch r.Method {
+		case http.MethodGet:
+			s := p.restAPIItems(r.URL.Path, -1)
+			if len(s) == 2 {
+				r.URL.Path = "/" + Rversion + "/" + Robjects + r.URL.Path
+				p.httpobjget(w, r)
+			} else if len(s) == 1 {
+				r.URL.Path = "/" + Rversion + "/" + Rbuckets + r.URL.Path
+				p.httpbckget(w, r)
+			} else {
```

Comment:
use 'fallthrough' here and in default?

Reply:
Can't `fallthrough` in the `else` or in a `default`, but I can `return` early in the first two cases and `fallthrough` otherwise

Reply:
us eat if possible, I was thinking return and fallthrough, see if the code looks good and decide

Reply:
may be don't use switch for now at all because there is only one case